### ISSUE TITLE
Overmind-devtools quits immediately without error

### ISSUE THREAD

**Issue by jadbox** [cerebral/overmind#435]
Version: "overmind-devtools": "^26.0.1"

When I run `npx overmind-devtools`, it just immediately quits without any error:

```
collabtube git:(master) ✗ npx overmind-devtools
collabtube git:(master)
```

Or when I run it with `concurrently`:

```
$ concurrently overmind-devtools next
[0] overmind-devtools exited with code 0
[1] ready - started server on http://localhost:3000
[1] info  - Using external babel configuration from /home/jdunlap/github/abridged/collabtube/.babelrc
[1] event - compiled successfully
```

**Comment by JeroenReumkens** [comment]
I have the same issue. I'm also struggling getting devtools to work for the last few weeks. The devtools in Vscode does connect, but only has the "initial state" and won't update. Perhaps this is related, so following this issue.

**Comment by christianalfoni** [comment]
I believe this is fixed with 27.0, closing now, but please reopen if still an issue
